src/index_runner/es_indexers/sample_set.py

```diff
@@ -17,6 +17,25 @@
 # _VER_SAMPLE_INDEX_NAME = 'sample_version_' + str(_SAMPLE_INDEX_VERSION)
 
 
+def _get_document(sample_document_id):
+    """ Get Sample document (if it exists) from Elasticsearch, otherwise return None
+    sample_document_id - "SMP::kbase_sample_id:kbase_sample_version"
+    """
+    es_url = config()['elasticsearch_url']
+    prefix = config()['elasticsearch_index_prefix']
+    es_url += f"/{prefix}.sample/_doc/{sample_document_id}"
+    resp = requests.get(url=es_url)
+    # if not resp.ok:
```

Comment:
Prob want to keep this?

Reply:
got rid of it because elastic doesn't '200' if the document doesn't exist.